### DISCUSSION TITLE
Email address in contact form

### DISCUSSION THREAD

**Discussion by sr258** [WeblateOrg/weblate#5798] [Q&A]
Email providers have become more strict when it comes to sending emails with email addresses that don't match the account's name. Our company's email provider rejects sending emails from names that don't match the account's address. The result is that the Weblate's contact form doesn't work anymore as it tries to put in the user's email address. Can you customize this behavior so that it always uses the service account's address?

**Comment by orangesunny** [comment]
Hi @sr258,

That’s a good point. An option to set these up will be added in Weblate 4.6. Follow https://github.com/WeblateOrg/weblate/issues/5812 for details.
